templates/includes/sidebar.html

```diff
@@ -1,4 +1,18 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}  
+		<li>
+      User: <a href="{% url "taxi:driver-detail" pk=user.id %}">{{ user.username }}</a>
+		</li>
+		<li>
+			<a href="{% url 'logout' %}">Logout</a>
+		</li>
+	{% else %}
+		<li>
+			<a href="{% url 'login' %}">Login</a>
+		</li>
+	{% endif %}
+  <hr>
+	<br>
```

Comment:
interestingly, I don't know why it is displayed like that here, because everything looks fine in PyCharm. I will write this part from scratch
![Screenshot from 2024-09-14 14-47-45](https://github.com/user-attachments/assets/6550854d-5a9d-4be4-a66e-4890fd16f76d)